buf/registry/module/v1beta1/module_service.proto

```diff
@@ -135,6 +135,8 @@ message CreateModulesRequest {
     // The name of the default Label of the Module.
     //
     // If not set, the default Label will be named "main" upon creation.
+    //
+    // This may not point to an archived Label.
```

Comment:
@doriable didn't we decide that default_label_name CAN point to an archived label?

https://github.com/bufbuild/registry-proto/pull/53/commits/07bf7b6f8820f872019e1d675c4deda101ab6e26

Reply:
Oh, not sure how this got revert :S let me add it back.